onnx_xla/backend.cc

```diff
@@ -48,7 +48,7 @@ namespace onnx_xla {
       case ONNX_NAMESPACE::TensorProto_DataType_STRING:			       \
       case ONNX_NAMESPACE::TensorProto_DataType_UNDEFINED:		       \
       default:  {							       \
-        throw conversion_error("Tensor not of a convertible data type.");      \
+        throw("Tensor not of a convertible data type.");                       \
```

Comment:
what's the reason of changing it to not throw a specific error type here?

Reply:
My reasoning was that ONNXIFI catches all exceptions and just outputs the message, so I thought generic error would be more concise. Should I make different error types for internal debugging purposes?

Reply:
Throwing a string is not recommended. Better use typed exception object.